test/index.js

```diff
@@ -12,6 +12,8 @@ lab.experiment('Web test', () => {
   })
 
   lab.test('GET / route works', async () => {
+    server.methods.api.getJson = (requestedUrl) => 1
```

Comment:
the 'getJson' hasn't been renamed 'get' causing the test to fail.
If a standard library rather than a plugin is used the implementation would have to change to use 'nock' to stub the http calls, or mocking the 'require' of the API library via Jest, or similar.

Reply:
Yep, good catch.
We are about to switch to Jest so I haven't looked into mocking with Lab.